test/Mono.Linker.Tests.Cases.Expectations/Metadata/DisableKeptItemsValidationAttribute.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Mono.Linker.Tests.Cases.Expectations.Metadata
+{
+	[AttributeUsage (AttributeTargets.Assembly, AllowMultiple = false)]
+	public class DisableKeptItemsValidationAttribute : BaseMetadataAttribute
```

Comment:
My preference would be to use `AttributeTargets.Class`.  This is how other test setup type attributes are defined today.  For example
```
[SkipPeVerify]
```

I think it's easiest to read the attributes on the test type when trying to understand what the test will do.

Also, putting this attribute at the assembly level would come with some annoyances.  For example, if multiple tests want to use this attribute they would need to #if theirs out and then use `[Define]` to include it when the test is compiled.

Now that I think about this more, what you are adding is really the same type of thing as `SkipPeVerifyAttribute`.  You are adding an attribute that disables the test frameworks default behavior.  So I would prefer to follow the same pattern as `SkipPeVerifyAttribute`.  This attribute should 

* Move to `Mono.Linker.Tests.Cases.Expectations.Assertions`
* Inherit from `BaseExpectedLinkedBehaviorAttribute`
* And have on it `[AttributeUsage (AttributeTargets.Class, AllowMultiple = false)]`

Reply:
@vitek-karas could you address the comments?

Reply:
Changed the attribute as per @mrvoorhe suggestion.